### ISSUE TITLE
Sensor is missing, but could still show the explanation

### ISSUE THREAD

**Issue by Noitarud** [phyphox/phyphox-android#68]
In the other experiments I can select "experiment info" in the elipsis (…) menu, but I was curious about the ones that are disabled - it only gives a message with a link to the "missing sensors" page. Maybe both messages can be shown on the same popup box.

**Comment by Staacks** [comment]
_First of all, thanks for all your input. I have seen a few from you pop up in my mail over the weekend and will now answer them all. However, please understand that I will be closing a few of them as this issue tracker (while not really well maintained to begin with) is more for issues (i.e. bugs, crashes, problems) and not that much about feature requests. We have a very long internal list of features that would be nice to have, but limited development capacities, so we usually decide based on how many users would benefit from something and the forums at https://phyphox.org/forums/ might be a much better place with more active users supporting a request or having additional input. I will also copy this block in italics to some of the issues that I close, so others who stumble upon them can read this, too._

I see how this might be interesting, especially for new users who might be wondering what they are missing out on. I will see if there is a simple way to offer this (not worth a massive redesign of the sensor not available dialog, though).

(Closing it as it is not really an issue, but I have put it on my "would be nice and could be done quickly" list, so you might see it in phyphox in a future release.)

**Comment by Noitarud** [comment]
Excellent 😉

**Comment by Staacks** [comment]
Just a quick update: This feature will probably not be included after all. We implemented it, but while testing we found that many of the description texts do not work well if the experiment cannot be opened as the longer ones often include instructions that are more confusing than helpful for most users that cannot open the experiment. The "sensor not available" dialog will mostly be seen by new users who do not yet have an idea what a gray experiment means and a simple and concice dialog is the better option here. So, we removed the code for this again (see https://github.com/phyphox/phyphox-ios/commit/dece940b4129ad72c38a799d233f31552bafca24).
